entries/q/qlik.com.json

```
@@ -0,0 +1,15 @@
+{
+  "Qlik": {
+    "domain": "qlik.com",
+    "additional-domains": [
+      "talend.com"
+    ],
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.qlik.com/en-US/cloud-services/Subsystems/Hub/Content/Sense_Hub/Introduction/qcs-MFA.htm",
+    "categories": [
+      "other"
```

Marketing and __Analytics__ might be a suitable category?
```suggestion
      "marketing"
```

---

I am not sure whether the **marketing** category includes data analysis websites. If you think it does, then I think it is fine! ❤️ @Carlgo11